config-bot/manifest.yaml

```diff
@@ -66,13 +66,13 @@ objects:
             - name: config-bot
               image: config-bot
               volumeMounts:
-                - name: github-token-mount
+                - name: github-coreosbot-token
                   mountPath: /var/run/secrets/coreos.fedoraproject.org/github-token
                   readOnly: true
           volumes:
-          - name: github-token-mount
+          - name: github-coreosbot-token
             secret:
-              secretName: config-bot-github-token
```

Comment:
Did we already (or plan to) invalidate this old one?

Reply:
It was actually the same token string, just under different secrets. They're both there in the old cluster; in the new cluster, they already share the same secret.